conjure_oxide/tests/integration/basic/comprehension/01/input.eprime

```diff
@@ -0,0 +1,16 @@
+language ESSENCE' 1.0
+
+find  m: matrix indexed by [int(1..5)] of int(1..5) 
+
+$ variable shadowing
```

Comment:
this is testing too many things at once, I fear. ok to keep this test case, but I'd like another one without variable shadowing too, so if one breaks (but not the other) we have a way of distinguishing what's broken.

Reply:
this could be the simpler example:

```
find  m: matrix indexed by [int(1..5)] of int(1..5) 
such that and([m[i] = i | i: int(1..5)])
```